streamer/pipeline_configuration.py

```diff
@@ -309,6 +319,19 @@ class PipelineConfig(configuration.Base):
                                    default=EncryptionConfig({})).cast()
   """Encryption settings."""
 
+  is_low_latency_dash = configuration.Field(bool, default=False).cast()
```

Comment:
design nit: I tend to think that "is" belongs on boolean getters, to see the state of a thing.  For a declaration like this to instruct the pipeline to do something, I would call it something like "low_latency_mode".  I would also make it apply to both DASH & HLS.

Reply:
Good point, I totally agree. I'll change it to "low_latency_dash_mode" for now since my Packager fork supports only LL-DASH. HOWEVER! :D I was tinkering this weekend and made promising progress for LL-HLS support! 🥳 I'll finish that up, and once LL-HLS is working, I'll comb over the naming/comments/logic to make sure everything is changed to general "low latency" rather than LL-DASH specific.